addons/account_peppol/static/src/components/res_config_settings_buttons/res_config_settings_buttons.js

```diff
@@ -41,7 +41,7 @@ class PeppolSettingsButtons extends Component {
     }
 
     get ediMode() {
-        return this.props.record.data.edi_mode;
+        return this.props.record.data.edi_mode || this.props.record.data.account_peppol_edi_mode;
```

Comment:
Not very very generic :(

Reply:
if only I didn't remove the prefix in the wizard :stuck_out_tongue_closed_eyes:

Reply:
@aliyatastemirova Sorry :cry: